### PR TITLE
SONARGRADL-17 Support SONARQUBE_SCANNER_PARAMS and sonar.scanner.skip

### PR DESCRIPTION
**Pull request by dbmeneses**

### CODE REVIEW

**Comment by henryju** [comment]
LGTM

File: src/main/java/org/sonarqube/gradle/SonarQubeTask.java [hunk 1]
```diff
@@ -19,6 +19,7 @@
  */
 package org.sonarqube.gradle;
 
+import java.util.HashMap;
```
Comment (SonarTech) [inline src/main/java/org/sonarqube/gradle/SonarQubeTask.java]: MINOR Remove this unused import 'java.util.HashMap'. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/sonarqube/gradle/SonarQubeTask.java [hunk 2]
```diff
@@ -86,6 +92,14 @@ public void run() {
     scanner.stop();
   }
 
+  private boolean isSkip(Properties properties) {
```
Comment (SonarTech) [inline src/main/java/org/sonarqube/gradle/SonarQubeTask.java]: MAJOR Make "isSkip" a "static" method. [rule](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS2325)

**Comment by SonarTech** [comment]
SonarQube analysis reported 2 issues
* MAJOR 1 major
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Pull request merged**